### PR TITLE
Jira v2 documentation fix

### PR DESCRIPTION
**Pull request by Noy-Maimon**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: demisto/etc#32521

## Description
fixing Jira documentation: missing info + bad formatting 

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

**Comment by Noy-Maimon** [comment]
@roysagi can you please have a look at the README file? Thank you

**Review by reutshal — APPROVED** [review]

File: Packs/Jira/Integrations/JiraV2/README.md [hunk 1]
```diff
@@ -14,21 +14,23 @@ For more information about manage syntax, see the https://support.atlassian.com/
 
 1. Navigate to __Settings__ > __Integrations__ > __Servers & Services__.
 2. Search for Jira v2.
-3. Click __Add instance__ to create and configure a new integration instance.
+3. __Authentiction__: As of June 2019, Basic authentication with passwords for Jira is no longer supported, please use an API Token or OAuth 1.0.
```
Comment (roysagi) [inline Packs/Jira/Integrations/JiraV2/README.md:17]:
```suggestion
3. __Authentiction__: As of June 2019, Basic authentication using passwords for Jira is no longer supported, please use an API Token or OAuth 1.0.
```

File: Packs/Jira/Integrations/JiraV2/README.md [hunk 2]
```diff
@@ -14,21 +14,23 @@ For more information about manage syntax, see the https://support.atlassian.com/
...
-Username__
-    * __Password (Deprecated - Use API token)__
-    * __API token__
+        * __Username__
+        * __Password (Deprecated)__
+        * __API token__
+        
     * ____________________ OAuth 1.0 __________________
+           
+      To use OAuth1.0 follow [this tutorial](https://developer.atlassian.com/cloud/jira/platform/jira-rest-api-oauth-authentication/) to get the Access Token. Authorizing using OAuth1.0, requires:
```
Comment (roysagi) [inline Packs/Jira/Integrations/JiraV2/README.md:29]: in step 3 you should add a note to look here as to how to get the API token.

File: Packs/Jira/ReleaseNotes/1_2_8.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Atlassian Jira v2
+- Documentation improvement.
```
Comment (roysagi) [inline Packs/Jira/ReleaseNotes/1_2_8.md:4]:
```suggestion
- Documentation and metadata improvements.
```

File: Packs/Jira/Integrations/JiraV2/JiraV2_description.md [hunk 4]
```diff
@@ -5,6 +5,7 @@ This will fetch all tickets in your system (including past tickets) that are in
 If you wish the first run to start from a specific time, use "Issue index to start fetching incidents from" param.
 
 Fetching incidents by creation time (using the Created field), instead of using IDs, is done by checking the "Use created field to fetch incidents" checkbox.
-To use OAuth1.0 follow [this tutorial](https://developer.atlassian.com/cloud/jira/platform/jira-rest-api-oauth-authentication/) to get the Access Token. Authorizing using OAuth1.0, requires Access Token, Private Key and Consumer Key.
 
-As of June 2019, Basic authentication with passwords for Jira is no longer supported, please use an API Token or OAuth 1.0
\ No newline at end of file
+As of June 2019, Basic authentication with passwords for Jira is no longer supported, please use basic authentication using API Token or use OAuth 1.0. To use basic authentication, follow [this tutorial](https://confluence.atlassian.com/cloud/api-tokens-938839638.html) to get the API token.
+To use OAuth1.0 follow [this tutorial](https://developer.atlassian.com/cloud/jira/platform/jira-rest-api-oauth-authentication/) to get the Access Token. Authorizing using OAuth1.0, requires Access Token, Private Key and Consumer Key.
```
Comment (roysagi) [inline Packs/Jira/Integrations/JiraV2/JiraV2_description.md:10]:
```suggestion

To use OAuth1.0 follow [this tutorial](https://developer.atlassian.com/cloud/jira/platform/jira-rest-api-oauth-authentication/) to get the Access Token. Authorizing using OAuth1.0, requires Access Token, Private Key, and Consumer Key. 
```

**Review by roysagi — APPROVED** [review]

**Review by glicht — APPROVED** [review]
looks good

**Comment by glicht** [comment]
@Noy-Maimon let's just add a note in what format we expect the key to be in.

**Comment by Noy-Maimon** [comment]
@glicht i already added the following format next to the key in brackets:

image

is this the key you meant?

**Comment by glicht** [comment]
> @glicht i already added the following format next to the key in brackets:
> 
> image
> 
> is this the key you meant? 

Ok. Didn't see this. This is sufficient.

**Pull request merged**